backend/de.metas.business/src/main/java/org/adempiere/mm/attributes/api/impl/AttributeSetInstanceBL.java

```diff
@@ -453,9 +453,35 @@ public AttributeSetInstanceId addAttributes(@NonNull final AddAttributesRequest
 		}
 
 		request.getAttributeInstanceBasicInfos().forEach(attributeValue -> {
-			setAttributeInstanceValue(asiId, attributeValue.getAttributeCode(), attributeValue.getValue() );
+			setAttributeInstanceValue(asiId, attributeValue.getAttributeCode(), attributeValue.getValue());
 		});
 
 		return asiId;
 	}
+
+	@Override
+	public boolean asiValuesMatchOrEmpty(@NonNull final AttributeSetInstanceId asiId1, @NonNull final AttributeSetInstanceId asiId2)
```

Comment:
@metas-rc  @adi-stefan  why not prefixing the method with `is`?
also, i think the name is a bit ambiguous/misleading. Maybe something like `isCommonAttributesMatching`...